capnp-conv/tests/capnp/test.capnp

```diff
@@ -0,0 +1,109 @@
+@0xc90daeac68e62b2a;
+
+struct TestStructInner {
+        innerU8 @0: UInt8;
+}
+
+struct TestUnion {
```

Comment:
Can this library handle structs that have some union fields and some non-union fields? E.g.

```capnp
struct Foo {
   a @0 :UInt64;
   b @1 :Text;
   union {
      c @2 :Data;
      d @3 :UInt32;
  }
}
```

Reply:
I am almost sure `capnp-conv` doesn't support this. What would be the Rust equivalent of this construct?